boa_engine/src/vm/opcode/get/property.rs

```diff
@@ -25,11 +26,66 @@ impl Operation for GetPropertyByName {
             value.to_object(context)?
         };
 
-        let key = context.vm.frame().code_block.names[index as usize]
-            .clone()
-            .into();
+        let ic = &context.vm.frame().code_block().ic[ic_index];
+        let mut slot = ic.slot();
+        if slot.is_cachable() {
+            let object_borrowed = object.borrow();
+            if ic.matches(object_borrowed.shape()) {
+                // println!(
+                //     "GET: T: \"{}\" {}: {:?}",
+                //     ic.name.to_std_string_escaped(),
+                //     slot.index,
+                //     slot.attributes
+                // );
```

Comment:
```suggestion

```

Reply:
Intentionally kept this, because it was very useful for debugging, maybe this is a good point to introduce tracing #2736 ?

Reply:
Removed the comment :)